diff --git a/dist/entrypoints/image-endpoint.js b/dist/entrypoints/image-endpoint.js
--- a/dist/entrypoints/image-endpoint.js
+++ b/dist/entrypoints/image-endpoint.js
@@ -1,4 +1,4 @@
-import { imageConfig } from "astro:assets";
+const imageConfig = undefined;
 import { isRemotePath } from "@astrojs/internal-helpers/path";
 import { isRemoteAllowed } from "@astrojs/internal-helpers/remote";
 const prerender = false;
diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -200,4 +200,2 @@
           vite.ssr.resolve.conditions.push("workerd", "worker");
-          vite.ssr.target = "webworker";
-          vite.ssr.noExternal = true;
           vite.build ||= {};
diff --git a/package.json b/package.json
--- a/package.json
+++ b/package.json
@@ -24,4 +24,5 @@
     "./image-service": "./dist/entrypoints/image-service.js",
     "./image-endpoint": "./dist/entrypoints/image-endpoint.js",
+    "./image-transform-endpoint": "./dist/entrypoints/image-endpoint.js",
     "./handler": "./dist/utils/handler.js",
     "./package.json": "./package.json"
